### ISSUE TITLE
top level async arrow function breaks use strict

### ISSUE THREAD

**Issue by monsanto** [babel/babel#828]
```
async () => {}
```

-->

```
var _this = this;
"use strict";

(function callee$0$0() {
  return regeneratorRuntime.async(function callee$0$0$(context$1$0) {
    while (1) switch (context$1$0.prev = context$1$0.next) {
      case 0:
      case "end":
        return context$1$0.stop();
    }
  }, null, _this);
});
```

use strict should be the first line

**Comment by sebmck** [comment]
`useStrict.wrap` needs to be added in the `aliasFunction` transformer probably.

**Comment by sebmck** [comment]
Fixed as of 4.4.0, thanks!
